### PR TITLE
adds arguments for various options

### PR DESCRIPTION
**Pull request by farfanoide**
adds -l to list all session files
adds -h to print usage
adds -o to open session file

### CODE REVIEW

File: tmuxstart [hunk 1]
```diff
@@ -1,41 +1,86 @@
-#!/bin/sh
+#!/usr/bin/env bash
```
Comment (treyhunner) [inline tmuxstart]: I would prefer not to rely on bash being installed.

File: tmuxstart [hunk 2]
```diff
@@ -1,41 +1,86 @@
...
+  esac
+done
 
 # Some helper functions for simpler session files
 new_session() {
-    # Set default path based on $path variable if defined
-    [ "$path" ] && cd "$path"
-    tmux new-session -d -s $session "$@"
-    [ "$path" ] && set_path "$path"
+  # Set default path based on $path variable if defined
```
Comment (treyhunner) [inline tmuxstart]: I'm purposely using 4 spaces for indentation as dictated by the `.editorconfig` file ([here](https://github.com/treyhunner/tmuxstart/blob/master/.editorconfig)).  See http://EditorConfig.org for more details.

**Comment by treyhunner** [comment]
Thanks for the pull request @farfanoide!

I like the idea of `-l` in particular.

I added a couple regarding code style.

**Comment by farfanoide** [comment]
i'll check the if statements in order to support sh. using bash makes everything a little easier  and it's normally available.
will update readme as well.
zsh tab completion on the way, will keep you posted ;)

**Comment by farfanoide** [comment]
also, my "send_command" is basically the same as "send_line" in https://github.com/treyhunner/tmuxstart/pull/7

**Comment by farfanoide** [comment]
https://github.com/farfanoide/tmuxstart/blob/completions/completions/_tmuxstart

**Comment by treyhunner** [comment]
I didn't realize that #7 had been updated.  I may be merging that soon

I think I like `send_line` better than `send_command`.  Since the "line" being sent could be `:wq <Enter>` to vim or something else ending in "Enter" that isn't necessarily a "command".  Thoughts on this?

**Comment by farfanoide** [comment]
+1 send_line, makes more sense.

**Comment by treyhunner** [comment]
Merged pull request #7 and added an AUTHORS file.  Remember to add yourself to it (by whatever information you prefer).

**Comment by farfanoide** [comment]
closing this to open separate pull requests for each argument.

**Pull request closed without merging**